index.bs

```diff
@@ -73,47 +73,42 @@ spec: webidl
+    This document specifies an API to query and request permissions to
+    powerful features on the Web platform. Web APIs have different ways to
+    deal with permissions. The [[Notifications]] API allows developers to
     request a permission and check the permission status explicitly. Others
-    might only expose the status to web pages when they try to use the API,
-    like the [[geolocation-API]] which fails if the permission was not
+    expose the status to web pages when they try to use the API,
+    like the [[Geolocation-API]] which fails if the permission was not
     granted without allowing the developer to check beforehand.
   </p>
   <p>
-    Being able to know whether an API call is going to prompt is useful in
-    order to provide a good user experience. Unfortunately, more often than
-    not, those prompts can't be controlled by developers.
+    The Permissions API provides the tools that potentially allow developers to
+    improve the user experience of web applications that require
+    multiple permission requests.
   </p>
   <p>
-    The API specified in this document is meant to provide the tools so
-    that web applications can improve their user experience when
-    permissions are involved.
+    The solution described in this document is meant to be extensible, but isn't
+    meant to be applicable to all the current and future permissions
+    available in the web platform. Working Groups that are creating specifications
+    whose permission model don't fit in the model described in this document
+    should contact the editors by
+    <a href="https://github.com/w3c/permissions/issues">filing an issue</a>.
   </p>
-  <p>
-    The solution described in this document is meant to be extensible but
-    isn't meant to be applicable to all the current and future permissions
-    available in the web platform. If you are working on a specification
-    that has a permission model that wouldn't fit in the model described in
-    this document, please contact the editors or file an issue. We would
-    love to hear about it.
-  </p>
-</section>
 <section class='non-normative'>
   <h2 id="privacy-considerations">
     Privacy considerations
   </h2>
   <p><em>This section is non-normative.</em></p>
   <p>
-    Permission states can be used as an element of fingerprinting by
-    websites. Usually websites could already have access to the information
-    but often through actually using the API which could lead to a
-    permission request UI if the permission was not already granted. Thus,
-    even though this API doesn't expose new fingerprinting data to
-    websites, it makes it easier for them to have discreet access to it.
-    Therefore, implementations are encouraged to have an option for users
-    to block (globally or selectively) the querying of permission states.
+    Implementations are encouraged to have an option for users to block (globally
```

Comment:
I liked putting this encouragement at the end better. Start with the privacy consideration, and end with a possible mitigation.